index.js

```diff
@@ -1,107 +1,76 @@
 /* eslint-disable no-console */
 const util = require('util');
 const { Executor } = require('@notify-watcher/executor');
-const githubNotificationsWatcher = require('./github-notifications');
-const gtdPlansWatcher = require('./gtd');
-const uniredTag = require('./unired-tag');
-const vtrPlansWatcher = require('./vtr');
+const githubNotificationsWatcher = require('./github-notifications'); // eslint-disable-line no-unused-vars
```

Comment:
```suggestion
require('./github-notifications');
```

That way you avoid doing the `// eslint-disable-line no-unused-vars`

Reply:
Although it seems you are using it below. Do you really need the disable?

Reply:
yes, that way if you want to check a watcher you just uncomment the line at the end